packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -124,15 +133,10 @@ calcite-link {
 }
 
 :host([scale="s"]) {
+  --calcite-internal-spacing: var(--calcite-spacing-sm);
```

Comment:
Should we include the component name in internal css variable names, like `--calcite-internal-tile-spacing`?  Not sure what would happen if another component defined a `--calcite-internal-spacing` var, but even though these are internal, might make sense to always include the component name even if just for findability in the codebase.

Reply:
Yeah, good catch. Will fix!